### PR TITLE
some what i consider careful improvements to the CoC after IRC discus…

### PR DESCRIPTION
**Pull request by hpk42**
…sions:

- substitute one "pledge" with "strive" but the headline is still
  "pledge".

- add "pleasurable" to "harassment-free" to not purely define things
  negatively

- add a "bad-day" paragraph to "Our standards"

- substitute internal references to "Code of Conduct" as i don't
  like the term. It's only the super-heading of everything
  because people came to expect to look for this term.

- substitute "enforcement" with "reporting and responses"
  as enforcement has a very police-like sound to me
  which i think is inappropriate here.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
-size, disability, ethnicity, gender identity and expression, level of experience,
-nationality, personal appearance, race, religion, or sexual identity and
-orientation.
+contributors and maintainers strive to making participation in the Autocrypt
+community a pleasurable, harassment-free experience. 
 
 ## Our Standards
 
-Examples of behavior that contributes to creating a positive environment include:
+Examples of behavior that we think contributes to creating a positive environment include:
```
Comment (moba) [inline CODE_OF_CONDUCT.md]: "we believe"?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -29,17 +26,22 @@ Examples of unacceptable behavior include:
...
-to witnessed instances of unacceptable behavior.
-
+Each maintainer is asked to take responsibility and appropriate, careful action 
+in response to witnessed instances of questionable behavior.
 Project maintainers have the right and responsibility to remove, edit, or
 reject comments, commits, code, wiki edits, issues, and other contributions
-that are not aligned to this Code of Conduct, or to ban temporarily or
-permanently any contributor for other behaviors that they deem inappropriate,
-threatening, offensive, or harmful.
+that violate "Our Standards" above.
```
Comment (moba) [inline CODE_OF_CONDUCT.md]: I liked the "are not aligned to this Code of Conduct" (without the rest of the sentence)

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (hpk42) [reply CODE_OF_CONDUCT.md]: just stroke the "we X", it's not really needed i guess.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (hpk42) [reply CODE_OF_CONDUCT.md]: k, reverted it.

**Comment by azul** [comment]
@moba is happy with this and so am i... So i just merged. It's a pr to the pr after all.

**Pull request merged**